backend/app/services/device/runtime_route.py

```
@@ -0,0 +1,237 @@
+    submitted_device_id: str,
+) -> RuntimeRouteIdentity | None:
+    """Resolve a logical or legacy Runtime ID within one user's active devices."""
+
+    devices = (
+        db.query(Kind)
+        .filter(
+            and_(
+                Kind.user_id == user_id,
+                Kind.kind == "Device",
+                Kind.namespace == "default",
+                Kind.is_active == True,
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)